### PR TITLE
Fix output stage for c2d.

### LINKED ISSUE #536: Create default output stage
Create default output stage

### PR DESCRIPTION
**Pull request by mariacarmina**
Fixes #536  .

Changes proposed in this PR:

- 
- 
-

### CODE REVIEW

File: src/components/c2d/compute_engines.ts [hunk 1]
```diff
@@ -217,6 +218,14 @@ export class C2DEngineOPFK8 extends C2DEngine {
         })
       index++
     }
+    let getOuput = {}
+    if (output) {
+      getOuput = output
+    } else if (config.hasHttp) {
+      getOuput = {
+        metadataUri: `${config.c2dNodeUri}:${config.httpPort}`
```
Comment (alexcos20) [inline src/components/c2d/compute_engines.ts:226]: please remove httpPort, as the doc says
```
export C2D_NODE_URI='http://127.0.0.1:8081' #for example
```
which already includes the port

**Review by alexcos20 — CHANGES_REQUESTED** [review]

File: src/components/c2d/compute_engines.ts (hunk 1 above)
Comment (mariacarmina) [reply src/components/c2d/compute_engines.ts:226]: Ok, I'll include port as well in URI everywhere

**Review by alexcos20 — APPROVED** [review]

**Comment by alexcos20** [comment]
Unfortunately, I will rewrite this PR, as it was based from develop after I screwed it up :(

**Comment by mariacarmina** [comment]
I'll close this and create a new one from main branch.

**Pull request closed without merging**